qutebrowser/utils/utils.py

```diff
@@ -915,3 +917,32 @@ def yaml_dump(data, f=None):
         return None
     else:
         return yaml_data.decode('utf-8')
+
+
+def pastebin_version():
+    """Pastebins version and logs to messages"""
+
+    app = qutebrowser.utils.objreg.get('app')
+    http_client = httpclient.HTTPClient()
+
+    def _get_paste_title():
+        return "qute version info {}".format(qutebrowser.__version__)
+
+    @pyqtSlot(str)
+    def _on_paste_version_success(url):
+        qutebrowser.utils.message.info("Version info pastebinned"
+                                       " to: {}".format(url))
+
+    @pyqtSlot(str)
+    def _on_paste_version_err(text):
+        qutebrowser.utils.message.info("Failed to pastebin version"
+                                       " info: {}".format(text))
+
+    pbclient = pastebin.PastebinClient(http_client, parent=app,
```

Comment:
Setting `parent=app` means this will never get cleaned up. You should probably call `pbclient.deleteLater()` in the success/error handlers.